web_tree_dynamic_colored_field/demo/res_users.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_users_tree" model="ir.ui.view">
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.view_users_tree" />
+        <field name="arch" type="xml">
+          <tree position="attributes">
+            <attribute name="colors">color_field: lang</attribute>
+          </tree>
+          <field name="login_date" position="attributes">
+            <attribute name="options">{
+              "bg_color": "#9e1635: login_date == False",
+              "fg_color": "white: login_date == False"
```

Comment:
I just tested locally and this view is installed with demo but no change to the background color happened:
![image](https://user-images.githubusercontent.com/23449160/55394589-f5675000-553f-11e9-995f-672b0f1980b9.png)